src/main/webapp/ts/matches.ts

```diff
@@ -41,9 +43,11 @@ function createMatchElement(match: User) {
     matchElement.className = 'list-group-item';
 
     const emailElement = document.createElement("a");
-    emailElement.innerText = match.email;
-    emailElement.href = "mailto:" + match.email;
-    emailElement.target = "_blank";
+    if(match.email != null) {
+        emailElement.innerText = match.email;
+        emailElement.href = "mailto:" + match.email;
+        emailElement.target = "_blank";
+    }
```

Comment:
Maybe use a sentinel value (e.g.: "<none provided>") so users understand why the link is blank (instead of blaming the system).

Reply:
Good idea, I added this